app-text/poppler/poppler23-23.09.0.recipe

```diff
@@ -365,6 +373,10 @@ REQUIRES_qt5="
 	lib:libQt5Xml$secondaryArchSuffix
 	lib:libtiff$secondaryArchSuffix
 	lib:libz$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libQt5Xml$secondaryArchSuffix
```

Comment:
These probably can be left out (same for Qt6), checking pkg-config --cflags for it reveals no extras.